### ISSUE TITLE
A11y_mssql for VSCode_Message Window_ScreenReader:Screen reader is not announcing the header and row reference for the data of row 1 time and the text besides it "Started Executing..."  in scan mode, 

### ISSUE THREAD

**Issue by prabhudevu** [microsoft/vscode-mssql#17450]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#BM_mssql for VSCode_Sep2022;#mssql for VSCode;#Win32;#DesktopApp;#A11ySev2;#AINotInScope;#WCAG4.2.3;#NVDA;#Jaws;#Win11;#Closed;

## Environment Details:
Application Name: mssql for VSCode
Windows Version: Win11
Visual studio code-insiders Version: 1.72.0-insider (user setup)
Screen Reader: NVDA:2022.1, JAWS: 2022.2112.24 ILM

## Repro Steps:

1. **Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix**
2. Open Visual studio code
3. Tab till extension button and search for "mssql"
4. Tab till install and move to the SQL Server button
5. Tab till Add Connection Button and insert the valid credentials
6. Right click on the server/database node you want to run the new query on.
7. Press enter “New Query” write any valid query and click on "Execute query button"
8. Message Window will open
9. Run NVDA and tab to the link "line" in message and check if the NVDA is announcing the message as table.

## Actual:
Screen reader is not announcing the header and row reference for the data of row 1 time and the text besides it "Started Executing..."  in scan mode as well as with table short cut keys (Ctrl + Alt + Arrow Keys).
There are no visual column header present for the rows

**Note:** Similar issue is observed with Jaws

## Expected:
The "time" and the text besides it "Started Executing..." Should be read in table format with appropriate row column number and header reference when navigated in scan mode or table short cut keys (Ctrl + Alt + Arrow Keys).

## User Impact:
Screen reader users will not get the proper information about the Data in table structure

## Attachments
![17450_A11y_mssql for VSCode_Message Window_ScreenReader](https://user-images.githubusercontent.com/98821792/190992347-4385ef33-2fbb-4f5c-90b5-55ac696fc2c5.gif)

![17450_A11y_mssql for VSCode_Message Window_ScreenReader](https://user-images.githubusercontent.com/98821792/191511581-55f29c9f-d99a-4969-805d-16760c749367.gif)

![17450_Closed](https://user-images.githubusercontent.com/98821792/203259403-eca8dd31-a48b-4a72-9aa8-8a4ede76d39b.png)

**Comment by prabhudevu** [comment]
Verified the issue in the latest Environment of MSSQL 1.1.7.0 issue is fixed, hence closing the bug. Added attachment for reference
